sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -510,6 +510,46 @@ def test_uniform_targets():
             assert_array_equal(model.alphas_, [np.finfo(float).resolution] * 3)
 
 
+@pytest.mark.filterwarnings("error::sklearn.exceptions.ConvergenceWarning")
```

Comment:
Do we need this? From the comment below it looks like by using a high number of iterations we reach convergence.